keyboards/Emi20/readme.md

```diff
@@ -0,0 +1,24 @@
+# Emi20
+
+![Emi20](https://i.imgur.com/Tt9ogmW.jpg)
```

Comment:
```suggestion
![Emi20](https://i.imgur.com/Tt9ogmWl.jpg)
```

Reply:
Why would I need to add the 1 to the link? Is this for if I would for example want to add more images?

Reply:
It's a lowercase L, to make imgur serve it at a smaller size. The original image is 3240x2340, this will make it a more reasonable 640x462.

Reply:
Ooh that makes more sense 😅